frontend/packages/dev-console/src/components/user-preferences/SecureRouteFields.scss

```diff
@@ -0,0 +1,3 @@
+.odc-secure-route-fields__select-fields {
+  max-width: 430.75px;
+}
```

Comment:
I don't know if we do this on all user preferences pages similarly, but we should avoid such fixed magic numbers. If we use this somewhere already we should update that as well.

If we don't want full width for forms we should use PatternFly form limit width feature.

https://www.patternfly.org/v4/components/form#limit-width